src/Bridges/indicatorbridge.jl

```diff
@@ -0,0 +1,57 @@
+"""
+    IndicatorActiveOnFalseBridge{T}
+
+The `IndicatorActiveOnFalseBridge` replaces an indicator constraint activated
+on 0 with a variable ``z_0`` with the constraint activated on 1, with a variable ``z_1``.
+It stores the added `variable_index` and added constraints:
+- ``z_1 \\in \\mathbb{B}`` in `zero_one_cons`
+- ``z_0 + z_1 == 1`` in `` in `disjunction_cons`
+- The added `ACTIVATE_ON_ONE` indicator constraint in `indicator_cons_index`.
+"""
+struct IndicatorActiveOnFalseBridge{T, F <: MOI.AbstractVectorFunction, S <: MOI.AbstractScalarSet} <: AbstractBridge
+    variable_index::MOI.VariableIndex
+    zero_one_cons::MOI.ConstraintIndex{MOI.SingleVariable, MOI.ZeroOne}
+    disjunction_cons::MOI.ConstraintIndex{MOI.ScalarAffineFunction{T}, MOI.EqualTo{T}}
+    indicator_cons_index::MOI.ConstraintIndex{F, MOI.IndicatorSet{MOI.ACTIVATE_ON_ONE, S}}
+end
+
+function bridge_constraint(::Type{<:IndicatorActiveOnFalseBridge}, model::MOI.ModelLike, f::MOI.VectorAffineFunction{T}, s::IS) where {S <: MOI.AbstractScalarSet, T <: Real, F, IS <: MOI.IndicatorSet{MOI.ACTIVATE_ON_ZERO, S}}
+    z1 = f.terms[1].scalar_term.variable_index
+    z2 = MOI.add_variable(model)
+    zo_cons = MOI.add_constraint(model, MOI.SingleVariable(z2), MOI.ZeroOne())
+
+    # z1 + z2 == 1
+    dcons = MOI.add_constraint(model,
+        MOI.ScalarAffineFunction(
+            [MOI.ScalarAffineTerm(one(T), z1), MOI.ScalarAffineTerm(one(T), z2)], zero(T),
+        ),
+        MOI.EqualTo(one(T)),
+    )
+    vec_terms2 = [t for t in f.terms if t.output_index == 2]
+    f2 = MOI.VectorAffineFunction(
+        vcat(
+            [MOI.VectorAffineTerm(1, MOI.ScalarAffineTerm(1.0, z2))],
+            vec_terms2,
+        ),
+        f.constants
+    )
+    ci = MOI.add_constraint(model, f2, MOI.IndicatorSet{MOI.ACTIVATE_ON_ONE}(s.set))
+
+    return IndicatorActiveOnFalseBridge{T, MOI.VectorAffineFunction{T}, S}(z2, zo_cons, dcons, ci)
+end
+
+function MOI.supports_constraint(::Type{<:IndicatorActiveOnFalseBridge},
+                                 ::Type{<:MOI.VectorAffineFunction},
+                                 ::Type{<:MOI.IndicatorSet{MOI.ACTIVATE_ON_ZERO}})
+    return true
+end
+
+function added_constraint_types(::Type{IndicatorActiveOnFalseBridge{T, F, S}}) where {T, F, S}
+    return [(F, MOI.IndicatorConstraint{MOI.ACTIVATE_ON_ONE, S})]
```

Comment:
This function and the one below are not tested

Reply:
I took example on the other bridges. How should we go about testing these I didn't find anything.
+ the functions themselves are pretty trivial they are all static (returning type-determined values)

Reply:
They are used by `LazyBridgeOptimizer`. You can a test in here:
https://github.com/JuliaOpt/MathOptInterface.jl/blob/6a4c3e1c3e00d4f30599bbe5e2d32c8f0286bad6/test/Bridges/lazybridgeoptimizer.jl#L165
to check that the indicator set activate on zero is supported by `full_bridge_optimizer` when it is not supported by the underlying model (but you need to add support for indicator set activate on one for the underlying model)